### DISCUSSION TITLE
AWS Lightsail prefered instance type?, how to export/import?

### DISCUSSION THREAD

**Discussion by fbcwebdev** [YOURLS/YOURLS#3073] [Q&A]
I see multiple sets of instructions for multiple types of instances that can be used.  My question is, what is best/simplest/preferred for this in AWS LightSail?

Also, what is the best way to get the site moved from old hosting.  I exported/imported the .sql file but the links in my new one don't show.

**Comment by dgw** [comment]
Instance size depends on how much load you expect and/or what other services you plan to run on the same Lightsail instance. My (non-Amazon) VPS with 2 GB of RAM is more than enough for my personal YOURLS, WordPress, IRC services, and several other things.

All I can say about migration is to make sure you match the database dump's table prefix, etc., or the new instance won't see the old data.

**Reply by fbcwebdev** [reply]
So not related to size, which type?  I'm trying nginx and it's over my head.  I've got a few wordpress instances that I understand.

**Reply by dgw** [reply]
Choice of HTTP server is entirely up to your comfort and ability to manage its configuration. Feel free to use whatever you already have running WordPress, which you say you understand. YOURLS and WP have nearly identical stacks (HTTPd + PHP + MySQL), so it should be fairly easy to adapt whatever you already use for WordPress even if the YOURLS wiki doesn't cover that server daemon.

**Reply by fbcwebdev** [reply]
Would I just make a new wordpress instance, but then rename the wordpress folder to YOURLS and put the files there?

**Reply by dgw** [reply]
Not quite. Though similar, YOURLS and WordPress use different filenames for things. If you want specific help you'll have to give specific details—most importantly which HTTP server you've chosen to use.

**Reply by fbcwebdev** [reply]
Which of these should I choose?  Nginx was a bust (404 errors that I cant figure out the configuration to), wordpress is overkill, LAMP next?
![Screen Shot 2021-09-23 at 2 43 52 PM](https://user-images.githubusercontent.com/70606593/134573571-3cec4072-0a10-4cf1-a7a5-5f996cc342c8.png)
?

**Reply by dgw** [reply]
LAMP will do nicely, and is quite popular. You can then configure it for YOURLS according to our [.htaccess wiki page](https://github.com/YOURLS/YOURLS/wiki/.htaccess).

**Reply by fbcwebdev** [reply]
Wow...the issues I'm running into!

Tried to use the info at https://linuxstans.com/how-to-install-yourls/ to do mysql_secure_installation and lost access.  More info in my bitnami post here: https://community.bitnami.com/t/mysql-root-not-working-after-running-mysql-secure-installation
